packages/tests/src/interfaces/IOrdersGatewayProxy.sol

```diff
@@ -0,0 +1,215 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
why are we duplicating this, means every time we need to change smth for the order gateway interface have to come back here and adjust ?

Reply:
This is almost automated, we don't do any copy paste from reya-network, but get ABI from the current deployment and use abi-to-sol to generate these interfaces. We should be able to generate these interfaces in a CLI command and AN and me are aware of this but we need some time to do it (especially that abi-to-sol has some limitations regarding events and custom datatypes).